apps/omg_watcher_rpc/lib/web/endpoint.ex

```diff
@@ -19,18 +19,20 @@ defmodule OMG.WatcherRPC.Web.Endpoint do
   plug(Plug.RequestId)
   plug(Plug.Logger, log: :debug)
 
+  if code_reloading? do
+    plug(Phoenix.CodeReloader)
+  end
+
   plug(
```

Comment:
Could we add a `: query_string_length` with a reasonable value (see. https://hexdocs.pm/plug/Plug.Parsers.html#module-options)?

Reply:
Since we're using a mish mush API (something between HTTP RPC and JSON-RPC) we can actually disable accepting `query_string_length `, or set it to 0. None of our APIs is accepting or expecting the params to come in the URL.

Reply:
I this case couldn't we just not copy `query_params` to `params` in `OMG.ChildChainRPC.Web.Plugs.MethodParamFilter` ?